apps/gauzy/src/app/pages/integrations/components/plugin-builtin/plugin-builtin.component.html

```
@@ -0,0 +1,102 @@
+@let integrations = integrations$ | async;
+@let isLoading = isLoading$ | async;
+
+<div class="builtin-container content-active first">
+	<!-- Search Bar -->
+	<div class="builtin-search">
+		<nb-form-field>
+			<nb-icon nbPrefix icon="search-outline"></nb-icon>
+			<input
+				#searchInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)